cardano-node/src/Cardano/Node/Configuration/Logging.hs

```diff
@@ -148,12 +153,12 @@ loggingCLIConfiguration = maybe emptyConfig readConfig
 
 -- | Create logging feature for `cardano-node`
 createLoggingLayer
-  :: Text
+  :: TraceOptions
```

Comment:
This seems suspicious. Why is this not part of the `NodeConfiguration`?

Reply:
In my understanding this has to do with the coexistence of the old and the new tracing system. So an abstraction in addition to NodeConfiguration makes these two levels easier to handle. But it can be removed, when the old tracing system will be abandoned, so it is only temporal.

Reply:
Okay, you are right, I have done this.